common/app/views/fragments/head.scala.html

```diff
@@ -15,7 +15,6 @@
 @* stylesheet <link>s - get the stylesheets downloading ASAP *@
 @fragments.stylesheets(projectName, getContent(page).exists(_.tags.isCrossword))
 
-<link rel="preload" as="script" href="@Static("javascripts/graun.standard.js")">
```

Comment:
yeah sorry simon, should have been WIP, just pushed a change that does that, will check CODE